agent/chain.py

```diff
@@ -64,52 +29,35 @@ def load_chains():
     """Logic for loading the chain you want to use should go here."""
     llm = ChatOpenAI(model_name = "gpt-4", temperature=1.2)
 
-    # chatGPT prompt formatting
-    objective_system_thought = SystemMessagePromptTemplate(prompt=OBJECTIVE_SYSTEM_THOUGHT)
-    objective_system_response = SystemMessagePromptTemplate(prompt=OBJECTIVE_SYSTEM_RESPONSE)
-
-    objective_human_thought = HumanMessagePromptTemplate(prompt=OBJECTIVE_HUMAN_THOUGHT)
-    objective_human_response = HumanMessagePromptTemplate(prompt=OBJECTIVE_HUMAN_RESPONSE)
-
-    objective_thought_chat_prompt = ChatPromptTemplate.from_messages([objective_system_thought, objective_human_thought])
-    objective_response_chat_prompt = ChatPromptTemplate.from_messages([objective_system_response, objective_human_response])
-
-    # define chains
-    objective_thought_chain = LLMChain(
-        llm=llm,
-        prompt=objective_thought_chat_prompt,
+    # define chain
+    bloom_chain = BloomChain(
```

Comment:
This is essentially a constructor for `BloomChain` with some pre-determined hardcoded inputs. I think generally the `agent/` folder should more be general interfaces and classes with concrete instances in the `bot/` and `www/` folders. We could make this a overloaded constructor within the class itself as a shortcut constructor.

TL;DR  - I think this should stay within the class instead of separate